authors/apps/articles/serializers.py

```diff
@@ -79,6 +79,7 @@ class Meta:
         read_only_fields = ['modified',
                             'author',
                             'slug']
+    """Get logged in user ID"""
```

Comment:
I'm not sure I understand the use of this

Reply:
That is part of Ken's work when he was working on Articles. It returns the ID of the user that's logged in.